tests/e2e/specs/shopper/checkout-place-order.test.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Internal dependencies
+ */
+import { shopper } from '../../../utils';
+import { SIMPLE_PRODUCT_NAME, BILLING_DETAILS } from '../../../utils/constants';
+
+if ( process.env.WOOCOMMERCE_BLOCKS_PHASE < 2 )
+	// eslint-disable-next-line jest/no-focused-tests
+	test.only( `Skipping checkout tests`, () => {} );
+
+describe( 'Shopper → Checkout → Can place an order', () => {
+	it( 'allows customer to place an order as a guest', async () => {
+		await shopper.logout();
+		await shopper.goToShop();
+		await shopper.addToCartFromShopPage( SIMPLE_PRODUCT_NAME );
+		await shopper.block.goToCheckout();
+		await shopper.block.fillBillingDetails( BILLING_DETAILS );
+		await shopper.block.placeOrder();
+		await expect( page ).toMatch( 'Your order has been received.' );
+	} );
+
+	it( 'allows customer to place an order as a logged in user', async () => {
+		await shopper.login();
+		await shopper.goToShop();
+		await shopper.addToCartFromShopPage( SIMPLE_PRODUCT_NAME );
+		await shopper.block.goToCheckout();
+		await shopper.block.placeOrder();
```

Comment:
The second test does fail on my end. We need to fill in the billing details here as well: `await shopper.block.fillBillingDetails( BILLING_DETAILS );`

Reply:
Done! 😀